### ISSUE TITLE
Settings UX: tab key does not properly switch focus

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52467]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : HighSierra10.13.5

**Additional Details:** 
MAS Violated : MAS 2.4.3

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown.
3. Focus order is not proper when navigating through all the lists,list items,buttons,links present on the screen while navigating using "Tab" key and also Shift Tab key.

**Actual:**
 Keyboard focus order is not properly aligned when navigating through the list items present on the page using Shift Tab key and Tab key 

When Tab key is pressed the focus order shifts from downward to upwards which is not the ideal functionality of the Tab key.

When Shift Tab key is pressed the focus order shifts from upwards  to downward which is not the ideal functionality of the Shift Tab key.

**Expected:**
 Keyboard focus order is should be properly aligned when navigating through the list items present on the page using Shift Tab key and Tab key 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to get clarity about focus order alignment and movement  when using Tab or Shift Tab key.

**MAS Reference:**
MAS 2.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment For Reference:**
[A11y_VSCode_UserSettings_Keyboard_FocusOrder.pptx](https://github.com/Microsoft/vscode/files/2119170/A11y_VSCode_UserSettings_Keyboard_FocusOrder.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is the new settings ux, assigning to @roblourens 
Feels like all those items needs to have proper tabindexes. Leaving for july - not sure when the settings  ux goes out of preview since that is the time we should address this issues.

**Comment by roblourens** [comment]
You say "focus order is not right", can you be more specific about what the problem is?

**Comment by isidorn** [comment]
@roblourens I think the focus is not jumping properly from left to right and from up to down. If I press tab a couple of times in the new settings ux the behavior is unexpected for me
Though @AccessibilityTestingTeam-TCS can explain what he exactly meant

**Comment by roblourens** [comment]
This is not the new settings editor, this is basically the same as #52468

**Comment by roblourens** [comment]
@isidorn you can weigh in on #54039 if you want

**Comment by isidorn** [comment]
@roblourens this is the new settings editor - it is quite clear if you look at the attachment.
Yes, let's treat all new settings editor focus jumping issues as part of #54039

**Comment by roblourens** [comment]
Are we looking at the same attachment? I see the old editor.

**Comment by isidorn** [comment]
My appologies, it is the old settings editor.

**Comment by isidorn** [comment]
@roblourens if I were you I would put this to the backlog. Since we now have the new settings experience
